osmaxx-py/excerptconverter/gisexcerptconverter/gis_excerpt_converter.py

```diff
@@ -0,0 +1,259 @@
+            #     'default': 'verbatim',
+            #     'values': [
+            #         {'name': 'verbatim', 'label': 'Verbatim'},
+            #         {'name': 'simplified', 'label': 'Simplified'}
+            #     ]
+            # }
+        }
+
+    @staticmethod
+    def extract_excerpts(execution_configuration, extraction_order, bbox_args, converter_helper):
+        """
+        Extract excerpt for chosen formats (execution_configuration) using docker-compose
+        to trigger the conversion process (defined in blackbox/docker-compose-conversion-blackbox.yml)
+
+        :param execution_configuration:
+        :param extraction_order:
+        :param bbox_args example: '8.775449276 47.1892350573 8.8901920319 47.2413633153'
+        :return:
+        """
+        for export_format_key, export_format_config in GisExcerptConverter.export_formats().items():
+            index = 0
+            if export_format_key in execution_configuration['formats']:
+                index += 1
+                extraction_command = "docker-compose run excerpt python excerpt.py %(bbox_args)s -f %(format)s" % {
+                    'bbox_args':  bbox_args,
+                    'format': export_format_key
+                }
+                subprocess.check_call(extraction_command.split(' '))
+
+                if len(os.listdir(settings.RESULT_MEDIA_ROOT)) > 0:
+                    for result_file_name in os.listdir(settings.RESULT_MEDIA_ROOT):
+                        # gis files are packaged in a zip file
+                        if GisExcerptConverter.create_output_file(extraction_order, result_file_name):
+                            converter_helper.inform_user(
+                                messages.SUCCESS,
+                                _('Extraction of "%(file_type)s" of extraction order "%(order_id)s" was successful. '
+                                  '(File %(file_index)s of %(number_of_files)s of %(converter_name)s converter)') % {
+                                    'file_type': export_format_config['name'],
+                                    'file_index': index,
+                                    'number_of_files': len(execution_configuration['formats']),
+                                    'converter_name': GisExcerptConverter.name(),
+                                    'order_id': extraction_order.id
+                                },
+                                email=False
+                            )
+                        else:
+                            converter_helper.inform_user(
+                                messages.ERROR,
+                                _('The extraction of "%(file)s" of extraction order "%(order_id)s" failed.') % {
+                                    'file': result_file_name,
+                                    'order_id': extraction_order.id
+                                },
+                                email=False
+                            )
+                else:
```

Comment:
It's hard to tell without counting or help from the IDE which `if` this `else` belongs to. The `if` block has much too many lines.